radio/src/gui/gui_common.cpp

```diff
@@ -780,6 +780,35 @@ int getFirstAvailable(int min, int max, IsValueAvailable isValueAvailable)
   return retval;
 }
 #if defined(MULTIMODULE)
+
+// This transforms OpenTX multi type to Multi type
+uint8_t convertMultiProtocole(uint8_t  moduleIdx, uint8_t type)
```

Comment:
This will need to be converted in 2.4, right?

Reply:
This will convert existing CUSTOM settings to regular settings. In 2.4, we need to re-align MULTI protocol list and otx multi protocol list (because sadly, yes, they are different, but that requires eeprom conversion)